backend/src/main/java/COMP_49X_our_search/backend/database/repositories/StudentRepository.java

```diff
@@ -0,0 +1,6 @@
+package COMP_49X_our_search.backend.database.repositories;
+
+import COMP_49X_our_search.backend.database.entities.Student;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+public interface StudentRepository extends JpaRepository<Student, Integer> {}
```

Comment:
we may want to add a method to findAllByMajors_Id, departments, etc like we did for projects so the faculty can filter by students too. But again, this might be something you are thinking of adding in a later iteration

Reply:
Yes, that's right, for now I'm only adding basic querying methods.